db/schema.rb

```diff
@@ -1251,13 +1251,13 @@
               END
             $function$
   SQL
-  create_function :update_phone_index_on_supporters, sql_definition: <<-SQL
+  create_function :update_phone_index_on_supporters, sql_definition: <<-'SQL'
       CREATE OR REPLACE FUNCTION public.update_phone_index_on_supporters()
        RETURNS trigger
        LANGUAGE plpgsql
       AS $function$
                   BEGIN
-                    new.phone_index = (regexp_replace(new.phone, '\\D','', 'g'));
+                    new.phone_index = (regexp_replace(new.phone, '\D','', 'g'));
```

Comment:
Oops :cry:

Reply:
That's correct now! It was always supposed to be a single `\` but before it was stripping out the first slash.